woocommerce/woocommerce-android
# [Simplified Login] Design update for illustration and "OR" divider design and color

This is a small PR that does the following:
- Move the divider layout into its own file for better reusabilty.
- Updates the colors of the line and text on the divider to match the design shared in p1667458224692009/1667281771.100359-slack-C046HDZL87J
- Updates illustration

Some learnings from this PR:
 I found that the login library already has an "or divider" layout however its colors are different from the suggested design above, so I chose to create our own layout instead. I also couldn't just reuse the code from the login library, because it doesn't work nicely with screen with different background colors. So the actual code uses my own implementation.


### Testing:
1. Run app in logged out mode,
2. Check divider prologue screen in light and dark mode,
3. Go to login by email and check divider in light and dark mode.

| Light mode | Dark mode |
|-|-|
|![Screenshot_20221104_162846](https://user-images.githubusercontent.com/266376/199939423-31790ba3-eab3-4d31-b64d-af1d17b2189a.png) | ![Screenshot_20221104_162903](https://user-images.githubusercontent.com/266376/199939410-e52ae35b-b416-4e5a-96fb-6ab17e0a1dfd.png) |
